catalog/view/theme/default/template/extension/payment/securesubmit.tpl

```diff
@@ -1,156 +1,327 @@
 <link rel="stylesheet" type="text/css" href="catalog/view/stylesheet/securesubmit.css">
-<form class="form-horizontal">
-    <fieldset id="payment">
-        <legend><?php echo $text_credit_card; ?><br>
 
-            <div class="ss-shield">  </div>
-            <div class="visa-gray hidden-xs "></div>
-            <div class="mc-gray hidden-xs"></div>
-            <div class="amex-gray hidden-xs"></div>
-            <div class="jcb-gray hidden-xs"></div>
-            <div class="disc-gray hidden-xs"></div>
+<!-- make iframes styled like other form -->
+<style type="text/css">
+    #iframes iframe{
+        float:left;
+        width:100%;
+    }
+    .iframeholder:after,
+    .iframeholder::after{
+        content:'';
+        display:block;
+        width:100%;
+        height:0px;
+        clear:both;
+        position:relative;
+    }
```

Comment:
Let's move these styles to our `securesubmit.css` file and make sure they are specific to our elements.

Reply:
styles moved to css file